extensions/AzureCosmosDBMongoDB/AzureCosmosDBMongoDB.FunctionalTests/appsettings.json

```diff
@@ -0,0 +1,71 @@
+{
+    "ServiceAuthorization": {
+        "Endpoint": "http://127.0.0.1:9001/",
+        "AccessKey": ""
+    },
+    "Logging": {
+        "LogLevel": {
+            "Default": "Trace",
+            "Microsoft.AspNetCore": "Trace"
+        }
+    },
+    "KernelMemory": {
+        "Services": {
+            "AzureCosmosDBMongoDB": {
```

Comment:
could you add this block also to `service/Service/appsettings.json`, under `Services`? (the list is sorted alphabetically).
Please add a comment for every line.
About the `ConnectionString` please leave it empty or add a template showing what the cs looks like, that will help users using the right value.